core/templates/dev/head/admin/Admin.js

```diff
@@ -56,10 +56,10 @@ oppia.controller('Admin', ['$scope', '$http', function($scope, $http) {
     $http.post($scope.adminHandlerUrl, {
       action: 'revert_config_property',
       config_property_id: configPropertyId
-    }).success(function() {
+    }).then(function() {
       $scope.message = 'Config property reverted successfully.';
       $scope.reloadConfigProperties();
-    }).error(function(errorResponse) {
+    }, function(errorResponse) {
       $scope.message = 'Server error: ' + errorResponse.error;
```

Comment:
Yep, I have checked that the porting of the .success methods do work manually. But I couldn't find a good way to test error responses since most of them are $post requests on Admin page (is there a page I could go to to check it? /admin doesn't seem to be it). Also, I thought the .error was a custom response added through interceptors since I couldn't find it on the deprecated [.error responses](https://code.angularjs.org/1.2.29/docs/api/ng/service/$http) either.